web/src/app/shared/components/projects-list/projects-list.component.ts

```diff
@@ -81,4 +83,8 @@ export class ProjectsListComponent implements OnChanges {
       return 'public_icon';
     }
   }
+
+  public delete(projectUid: string): void {
+    this.projectService.showDeleteDialog(projectUid);
```

Comment:
Having the `subscribe` here makes sense, we don't want to redirect here but show a `snack` message, because this component could be on multiple pages and the redirect could be to the wrong place